packages/vats/src/vat-bank.js

```diff
@@ -357,7 +376,7 @@ const prepareBank = (zone, { makePublishKit, makeVirtualPurse }) => {
     },
     {
       getAssetSubscription() {
-        return provideAssetSubscription(
+        return makeAssetSubscription(
```

Comment:
why return a fresh one each time? Isn't the state the same? the provide method seems more appropriate. Or if you like having the Exo maker,
```suggestion
        return produce(baggage, 'assetSubscription', makeAssetSubscription);
```

Reply:
I updated to `provideAssetSubscription` and tested it in the upgrade test.